controllers/ixiatg_controller.go

```diff
@@ -110,8 +110,8 @@ const (
 	IXIA_C_GRPC_VERSION   string = "0.0.1-3114"
 
 	LIVENESS_DELAY   int32 = 10
```

Comment:
delay should be set to 0 or close to 0 I believe.

If images are already loaded, containers can boot up instantly. Adding a delay of 10 seconds would make the "liveness" 10 seconds slower.

Reply:
Updated delay to 1 sec